# Set the gripper joint values to the values of the 'close' named target

---
bool success
